front/src/common/Map/Layers/GeoJSONs.tsx

```diff
@@ -371,6 +371,8 @@ const GeoJSONs: FC<{
   beforeId?: string;
   // When true, all layers are rendered (ie "minZoom" restrictions are ignored)
   renderAll?: boolean;
+  // Is used in the editor ?
```

Comment:
What do you mean by that ?

Reply:
Component `GeoJSON` is used at multiple places in the app.
For this feature, I need to replace some map style only for the editor., not everywhere.
So I added this prop to know if the component is used within the editor.

For now, the only diff in style is the text label with the ID, but perhaps in futur we will have more diff.  
That's why it's name `editor` and not `addIdInLabel` or something like that.

Reply:
Ok I was just disturbed with the "?". Maybe a proper sentence and explanation like the one you just gave me would be more understanding for a dev passing here